python/215-Kth-Largest-Element-in-an-Array.py

```diff
@@ -44,3 +44,15 @@ def findKthLargest(self, nums: List[int], k: int) -> int:
                 break
 
         return nums[k]
+ 
+#Heap Solution (For Reference)
```

Comment:
Thanks for contributing @sxsmg!, please add the space and time complexity to match the same format as the other solutions and rename this to `Solution3`.